doc/release-notes/rl-2103.adoc

```diff
@@ -72,6 +72,35 @@ programs.rofi.extraConfig = {
 };
 ----
 
+* The `services.redshift.extraConfig` and `services.gammastep.extraConfig`
+options were removed in favor of <<opt-services.redshift.settings>> and
+<<opt-services.gammastep.settings>>, that are now an attribute set rather
+than a string. They also support new features not available before, for
+example:
++
+[source,nix]
+----
+services.redshift = {
+  dawnTime = "6:00-7:45";
+  duskTime = "18:35-20:15";
+  settings = {
+    redshift = {
+      gamma = 0.8;
+      adjustment-method = "randr";
+    };
+
+    randr = {
+      screen = 0;
+    };
+  };
+};
+----
+
+It is recommended to check either
```

Comment:
```suggestion
----
+
It is recommended to check either
```

Otherwise the paragraph is on a standalone line and not under the bullet point (wrongly indented).